addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/AddOnJob.java

```diff
@@ -210,8 +210,8 @@ public void showDialog() {
     public String getSummary() {
         return Constant.messages.getString(
                 "automation.dialog.addon.summary",
-                this.getData().getInstall() == null ? "[]" : this.getData().getInstall(),
-                this.getData().getUninstall() == null ? "[]" : this.getData().getUninstall());
+                this.getData().getInstall().isEmpty() ? "[]" : this.getData().getInstall(),
```

Comment:
as get install and get uninstall are not returning null so i think too string of list will only suffice.